# Parameter for planet_landing.cpp

##沙盘高度
mark_hight: 0.2

#在沙盘中心时的高度
first_hight: 5.5

#在1/4沙盘区域时的高度
second_hight: 4

#飞到着陆点上方时的高度
third_hight: 1

##相机参数(视角为90°，1280*720)
fx: 523.2
fy: 523.2
cx: 640.0
cy: 360.0

##畸变参数
k1: 0.0007689952035434544
k2: 0.030314499512314796
p1: -0.028857899829745293
p2: 0.0037220960948616266
k3: 0.0


